Jenkinsfile_CNP

```diff
@@ -6,9 +6,16 @@ import uk.gov.hmcts.contino.Environment
 
 Environment environment = new Environment(env)
 
+slackChannel = "#ccd-master-builds"
+
 onMaster {
-  enableSlackNotifications('#ccd-master-builds')
-  sharedInfrastructurePipeline('ccd', environment.nonProdName, 'nonprod')
-  sharedInfrastructurePipeline('ccd', environment.demoName, 'nonprod')
-  sharedInfrastructurePipeline('ccd', environment.prodName, 'prod')
+    try {
+        sharedInfrastructurePipeline('ccd', environment.nonProdName, 'nonprod')
+        sharedInfrastructurePipeline('ccd', environment.demoName, 'nonprod')
+        sharedInfrastructurePipeline('ccd', environment.prodName, 'prod')
+    } catch (err) {
+          notifyBuildFailure channel: slackChannel
+          throw err
+    }
+    notifyBuildFixed channel: slackChannel
```

Comment:
wouldn't it make more sense to have `notifyBuildFixed` as the last line inside the `try` block? that way you don't rely on the `catch` block rethrowing the error.

Reply:
I mostly mapped it to how that had been implemented, but does make it more readable where you say. I just re-throw'd cause I guessed Jenkins wanted to know.  I'll keep the throw and move the Fixed call.

Reply:
OK just did what you said :) Its more readable now, tx.